lib/zstd.h

```diff
@@ -1954,6 +1956,9 @@ ZSTDLIB_STATIC_API size_t ZSTD_CCtx_refPrefix_advanced(ZSTD_CCtx* cctx, const vo
  */
 #define ZSTD_c_deterministicRefPrefix ZSTD_c_experimentalParam15
 
+/* TODO document */
+#define ZSTD_c_assumeColdCDict ZSTD_c_experimentalParam16
```

Comment:
I think the name is a bit awkward, since this is a 3-way switch.

Maybe `ZSTD_c_coldDictMode`?